assets/pages/new-contributions-landing/components/StripePaymentRequestButton.jsx

```diff
@@ -160,7 +160,7 @@ function initialisePaymentRequest(props: PropTypes) {
     // We need to do this so that we can offer marketing permissions on the thank you page
     updateUserEmail(data, props.updateEmail);
     const tokenId = props.isTestUser ? 'tok_visa' : token.id;
-    props.onPaymentAuthorised({ paymentMethod: 'Stripe', token: tokenId })
+    props.onPaymentAuthorised({ paymentMethod: 'Stripe', token: tokenId, stripePaymentMethod: 'StripeApplePay' })
```

Comment:
Is the component in this file specifically for Apple pay?

Reply:
Is it currently. When we expand it to the Stripe Payment Request Button, we will have to update this.